src/lib/popover/_animations.scss

```diff
@@ -20,6 +30,16 @@
   }
 }
 
+@keyframes slideout {
```

Comment:
The animations aren't respecting prefers reduced motion. Not a big deal for fade in/out, but sliding and zooming should automatically disable.